src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -254,7 +254,7 @@
     //  "DefaultSender": "",
     //  "ConnectionString": ""
     //}
-    //"OrchardCore_Sms_AzureCommunicationServices": {
+    //"OrchardCore_Sms_AzureCommunication": {
```

Comment:
This will be breaking now

Reply:
not a breaking change since we have not yet release 2.1

Reply:
Oops I thought it was Azure Email :)